lib/widgets/dashboard/widgets.dart

```diff
@@ -309,6 +327,101 @@ class _DashboardWeightWidgetState extends State<DashboardWeightWidget> {
+        .map(
+          (item) => CategoriesCard(item),
+        )
+        .toList();
+    return Consumer<BodyWeightProvider>(
+      builder: (context, workoutProvider, child) => Card(
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            ListTile(
+              title: Text(
+                AppLocalizations.of(context).measurements,
+                style: Theme.of(context).textTheme.headline4,
+              ),
+              leading: const FaIcon(
+                FontAwesomeIcons.weight,
+                color: Colors.black,
+              ),
+            ),
+            Column(
+              children: [
+                if (items.isNotEmpty)
+                  Column(children: [
+                    CarouselSlider(
+                      items: items,
+                      carouselController: _controller,
+                      options: CarouselOptions(
+                          autoPlay: false,
+                          enlargeCenterPage: false,
+                          viewportFraction: 1,
+                          enableInfiniteScroll: false,
+                          aspectRatio: 1.1,
+                          onPageChanged: (index, reason) {
+                            setState(() {
+                              _current = index;
+                            });
+                          }),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.only(bottom: 10),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: items.asMap().entries.map((entry) {
+                          return GestureDetector(
+                            onTap: () => _controller.animateToPage(entry.key),
+                            child: Container(
+                              width: 12.0,
+                              height: 12.0,
+                              margin: EdgeInsets.symmetric(
+                                  vertical: 8.0, horizontal: 4.0),
+                              decoration: BoxDecoration(
+                                  shape: BoxShape.circle,
+                                  color: (Theme.of(context).brightness ==
+                                              Brightness.dark
+                                          ? Colors.white
+                                          : wgerPrimaryColor)
+                                      .withOpacity(
+                                          _current == entry.key ? 0.9 : 0.4)),
+                            ),
+                          );
+                        }).toList(),
+                      ),
+                    ),
+                  ])
+                else
+                  NothingFound(
+                    AppLocalizations.of(context).noWeightEntries,
+                    AppLocalizations.of(context).newEntry,
+                    WeightForm(),
```

Comment:
I'm not sure what to do here if there are no categories. From one side we could simply allow the user to add one, but it is then probably a bit strange that they can't add another one. It might be enough to just display some info text and add a button that takes the user to the measurement overview page

Reply:
Maybe there can be arrow -> (alligned on the right from title for example "Weights" or "Measurements") and on press function will be navigate to "Weights" or "Measurements" screens.

Reply:
something like this... 
![image](https://user-images.githubusercontent.com/62099703/194172206-320d65cb-0165-41d2-8110-bd7b40358464.png)

also weights tab should be changed as measurements (add icon wont be there but at same row as "go to detail page"
![image](https://user-images.githubusercontent.com/62099703/194172271-e3b7af58-7036-494a-bb7c-419bd0a2d84c.png)

Reply:
yes, in any case all cards should behave the same with respect to going to the detail page or adding entries